src/domain/flow-system/domain/src/entities/flow_configuration/flow_configuration_rule.rs

```diff
@@ -18,6 +18,7 @@ pub enum FlowConfigurationRule {
     Schedule(Schedule),
     BatchingRule(BatchingRule),
```

Comment:
If "schedule" becomes "ingest", then "batching" should probably become "transform".